threedi_api_client/api.py

```diff
@@ -85,14 +85,32 @@ def __init__(self, env_file=None, config=None, version="v3", asynchronous=False)
         else:
             user_config = EnvironConfig()
 
-        # Get the config variables
         host = user_config.get("THREEDI_API_HOST")
         username = user_config.get("THREEDI_API_USERNAME")
-        password = user_config.get("THREEDI_API_PASSWORD")
-        if not all(x for x in (host, username, password)):
+        if not all(x for x in (host, username)):
             raise ValueError(
                 "ThreediApi requires the THREEDI_API_HOST, THREEDI_API_USERNAME, "
-                "and THREEDI_API_PASSWORD configuration values."
+                "configuration values."
+            )
+
+        # Get the config variables
+        password = user_config.get("THREEDI_API_PASSWORD")
+        try:
+            access_token = user_config.get("THREEDI_API_ACCESS_TOKEN")
+        except KeyError:
+            access_token = None
+        try:
+            refresh_token = user_config.get("THREEDI_API_REFRESH_TOKEN")
+        except KeyError:
+            refresh_token = None
+
+        tokens = all(x for x in (access_token, refresh_token))
```

Comment:
Better to be more strict in user-facing API.

If you only supply one access and not a refresh token, raise that they both have to be given.
Also, if you supply both tokens and password, raise that you have to choose between one of the two authentication options.